src/amo/pages/Addon/index.js

```diff
@@ -611,7 +611,8 @@ export class AddonBase extends React.Component {
 }
 
 export function mapStateToProps(state, ownProps) {
-  const { slug } = ownProps.match.params;
+  let { slug = '' } = ownProps.match.params;
```

Comment:
I'm not sure that you need to default `slug` to `''` here. You're checking below that `slug` is a string before trying to call `trim()` on it, so I don't think you need to default it to anything. 

It's interesting that the [Flow definition of `getAddonBySlug`](https://github.com/mozilla/addons-frontend/blob/2018.11.08/src/core/reducers/addons.js#L319-L322) states that `slug` must be a string, but it is also checking internally for `typeof string`, so that suggests that `slug` could be something other than a string.